src/zope/exceptions/tests/test_exceptionformatter.py

```diff
@@ -668,7 +668,7 @@ def test_multiline_exception(self):
         self.assertEqual(lines[0], '    syntax error')
         # PyPy has a shorter prefix
         self.assertTrue(lines[1].endswith('    ^'))
-        self.assertEqual(lines[2], 'SyntaxError: invalid syntax')
+        self.assertRegex(lines[2], '^SyntaxError: invalid syntax')
```

Comment:
Python 2.7 doesn't have this, but it has `assertRegexpMatches`.  Python 3 also has `assertRegexpMatches`, but it's marked as deprecated.

Or we could do 
```suggestion
        self.assertTrue(lines[2].startswith('SyntaxError: invalid syntax'), lines[2])
```